contracts/examples/LiquidityExamples.sol

```diff
@@ -0,0 +1,238 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity =0.7.6;
+pragma abicoder v2;
+
+import '@uniswap/v3-core/contracts/interfaces/IUniswapV3Pool.sol';
+import '@uniswap/v3-core/contracts/libraries/TickMath.sol';
+import '@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol';
+import '../libraries/TransferHelper.sol';
+import '../interfaces/INonfungiblePositionManager.sol';
+import '../base/LiquidityManagement.sol';
+
+contract LiquidityExamples is IERC721Receiver {
+    address public constant DAI = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+    address public constant USDC = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+
+    uint24 public constant poolFee = 3000;
+
+    INonfungiblePositionManager public immutable nonfungiblePositionManager;
```

Comment:
bit odd for this to be the only immutable, i'd just hardcode all of DAI/USDC/NFT manager to be dummy addresses